pkg/controller/runtimecomponent/runtimecomponent_controller.go

```diff
@@ -322,6 +322,20 @@ func (r *ReconcileRuntimeComponent) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, nil
 	}
 
+	isApplicationSupported, err := r.IsGroupVersionSupported(servingv1alpha1.SchemeGroupVersion.String())
```

Comment:
I would suggest exploring option to move this to base reconciler, so it can be shared between controllers if possible